src/Plugin/Validators/ValidDelimitedFile.php

```diff
@@ -27,6 +27,69 @@ class ValidDelimitedFile extends TripalCultivateValidatorBase {
   use FileTypes;
   use ColumnCount;
 
+  /**
+   * A mapping of all of the tokens supported by this validator.
+   *
+   * @var array
+   *   An associative array mapping tokens to their details, such as the
+   *   developer case string and the default message to substitute the token.
+   *   The following tokens are implemented for this mapping, with the following
+   *   descriptions for their 'default-msg' values:
+   *   - 'table-unsupported': the token containing the message used by cases
+   *     where raw row is empty or raw row is not delimited.
+   *   - 'table-delimited': the token containing the message used by cases
+   *     where raw row has excess column or raw row has missing columns.
+   *   Tokens below cannot be overriden as their value is determined at runtime:
```

Comment:
Technically not true for the case tokens. How about:
```suggestion
   *   Tokens below cannot be overriden as their value is determined at runtime
   *   or case messages are designed to cover multiple cases. See above for
   *   tokens that can be changed.
```